### PR TITLE
How to remove old elasticsearch indices

### PR DESCRIPTION
**Pull request by leenagupte**
Instructions about how to find old elasticsearch indices that should have been deleted to regain disk space.

### CODE REVIEW

File: source/manual/alerts/low-available-disk-space.html.md [hunk 1]
```diff
@@ -102,6 +102,34 @@ Steps to investigate postgres db size:
 1. You can choose one of the dbs by doing: `\c <name-of-db>`
 For example: `\c email-alert-api_production` 
 
+## Low available disk space on /mnt/elasticsearch
+
+Usually this is caused by old indices not being closed.
+
+View the dashboard to see how many indices are active on the cluster:
+
+`ssh -L9200:localhost:9200 rummager-elasticsearch-1.api.staging`
```
Comment (kevindew) [inline source/manual/alerts/low-available-disk-space.html.md]: May want to change this to production?

File: source/manual/alerts/low-available-disk-space.html.md [hunk 2]
```diff
@@ -102,6 +102,34 @@ Steps to investigate postgres db size:
...
+
+Then visit http://localhost:9200/_plugin/head/
```
Comment (kevindew) [inline source/manual/alerts/low-available-disk-space.html.md]: Do you want to make this a link?

File: source/manual/alerts/low-available-disk-space.html.md [hunk 3]
```diff
@@ -102,6 +102,34 @@ Steps to investigate postgres db size:
...
+
+![Screenshot of a healthy elasticsearch dashboard](images/elasticsearch/elasticsearch-healthy-dashboard.png)
+
+If old indices haven't been deleted, you'll see duplicates.
+
+![Screenshot of an elasticsearch dashboard with duplicate indices](images/elasticsearch/elasticsearch-dashboard-with-duplicate-indices.png)
+
+The indices apart from the ones in use can be closed and then, if no problems arise, deleted.
+
+To close an index, select `Actions -> Close` and then wait a few minutes to make sure everything is ok. If there is a problem, the index can still be re-opened.
```
Comment (kevindew) [inline source/manual/alerts/low-available-disk-space.html.md]: For seeing if everything is ok you might want to suggest monitoring icinga for alerts.

File: source/manual/alerts/low-available-disk-space.html.md (hunk 1 above)
Comment (leenagupte) [reply source/manual/alerts/low-available-disk-space.html.md]: Sure.

**Comment by leenagupte** [comment]
@kevindew I've addressed your comments and fixed-up

File: source/manual/alerts/low-available-disk-space.html.md [hunk 4]
```diff
@@ -102,6 +102,34 @@ Steps to investigate postgres db size:
...
+
+![Screenshot of a healthy elasticsearch dashboard](images/elasticsearch/elasticsearch-healthy-dashboard.png)
+
+If old indices haven't been deleted, you'll see duplicates.
+
+![Screenshot of an elasticsearch dashboard with duplicate indices](images/elasticsearch/elasticsearch-dashboard-with-duplicate-indices.png)
+
+The indices apart from the ones in use can be closed and then, if no problems arise, deleted.
+
+To close an index, select `Actions -> Close` then wait a few minutes and monitor Icinga make sure everything is ok. If there is a problem, the index can still be re-opened.
```
Comment (kevindew) [inline source/manual/alerts/low-available-disk-space.html.md]: think we're just missing a "to" here.

**Comment by leenagupte** [comment]
@kevindew Fixed up again!

**Pull request merged**